src/inmanta/util/dict_path.py

```diff
@@ -620,6 +821,12 @@ def get_elements(self, container: object) -> list[object]:
         else:
             raise ContainerStructureException(f"{container} is not a Dict")
 
+    def get_paths(self, container: object) -> Sequence["NullPath"]:
+        if self._validate_container(container):
+            return [NullPath()]
```

Comment:
could it not return `[]` ?

Reply:
I am not sure.  An empty list here would mean "there is no path matching this filter in the give container", but here there is